SLAM/EKFSLAM/ekf_slam.py

```diff
@@ -30,9 +30,9 @@
 def ekf_slam(xEst, PEst, u, z):
     # Predict
     S = STATE_SIZE
```

Comment:
Can we remove S?

Reply:
Certainly, `S` is used to denote the innovation covariance in the update step, so using `STATE_SIZE` in the predict step enhances code clarity.